src/main/java/com/codesquad/rare/domain/post/PostController.java

```diff
@@ -0,0 +1,63 @@
+package com.codesquad.rare.domain.post;
+
+import static com.codesquad.rare.common.api.ApiResult.OK;
+
+import com.codesquad.rare.common.api.ApiResult;
+import com.codesquad.rare.domain.post.request.PostCreateRequest;
+import com.codesquad.rare.domain.post.response.PostCreateResponse;
+import java.util.List;
+import javax.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("posts")
+public class PostController {
+
+  private final PostService postService;
+  private static final String DEFAULT_PAGE = "0";
+  private static final String DEFAULT_SIZE = "20";
+
+  //생성시간 순으로 내림차순 정렬
+  @GetMapping("createdAt")
+  public ApiResult<List<Post>> findAllInLatestOrder(
+      @RequestParam(value = "page", required = false, defaultValue = DEFAULT_PAGE) int page,
+      @RequestParam(value = "size", required = false, defaultValue = DEFAULT_SIZE) int size) {
+    return OK(postService.findAllAndOrderByCreatedAtDesc(page, size));
```

Comment:
Service 단의 메소드 이름이 모두 너무 길다고 생각합니다.
아마 Repository 의 것을 그대로 차용한 것으로 보이는데.. 기능이 많아질수록 메소드를 읽는데 걸리는 시간이 너무 오래 걸릴 듯 합니다.

Service 단의 메소드를 좀 더 단순화 시키고 파라미터를 받는 형태를 추천합니다.
아래는 하나의 예시입니다.

```suggestion
    return OK(postService. findAll(page, size, reverseOrder));
```

Reply:
좋은 방향이네요!
차후 리팩토링 진행시, 서비스 메서드를 좀 더 깔끔하게, 그리고 파라미터를 통해 결과값이 조절되도록 만들어보겠습니다 
감사합니당 ㅎㅎ